src/components/home/about_section/team_subsection.js

```diff
@@ -7,7 +7,7 @@ import styles from "./team_subsection.module.css"
 
 const teamQuery = graphql`
   query TeamQuery {
-    allTeamYaml {
+    allTeamYaml(filter: { active: { ne: false } }) {
```

Comment:
I think thaving `active == true` would be more readable no? Just my opinion anyway, feel free to merge 👍

Reply:
If it works, I'm a bit noobish with GraphQL, I usually follow along GraphiQL. Lemme see

Reply:
Apparently that's not a thing for Gatsby :man_shrugging:

Reply:
`filter: { active: { eq: true } }` did not work? Weird

Reply:
Oh, I thought you actually were saying to use a different syntax like `filter: { active == true }`.

`filter: { active: { eq: true } }` works but doesn't do what we want here. In this specific case, and to avoid adding unnecessary fields to the data file, we want to exclude all those team members for which the `active` field is explicitly `false`. If unset, we'll default to assuming the team member is active. If we ask GraphQL for the team members that have the `active` field with `true`, none would be shown.

Reply:
Got it :+1: